filesystems/s3/src/main/java/org/lerch/s3fs/S3Channel.java

```diff
@@ -0,0 +1,17 @@
+package org.lerch.s3fs;
```

Comment:
If we're making changes here we'll also want to make a PR to [emil's repo](https://github.com/elerch/Amazon-S3-FileSystem-NIO2)

Reply:
Thanks for the tip, will do

Reply:
https://github.com/Upplication/Amazon-S3-FileSystem-NIO2/pull/125

Reply:
https://github.com/elerch/Amazon-S3-FileSystem-NIO2/pull/2